docs/API.md

```diff
@@ -889,13 +889,28 @@ const Timer = enhance(({ timeElapsed }) =>
 
 ```js
 createEventHandler<T>(): {
-  handler: (value: T) => void
-  stream: Observable<T>,
+  handler: (value: T) => void,
+  stream: Observable<T>
 }
 ```
 
 Returns an object with properties `handler` and `stream`. `stream` is an observable sequence, and `handler` is a function that pushes new values onto the sequence. Useful for creating event handlers like `onClick`.
 
+### `createEventHandlerWithConfig()`
```

Comment:
Could you also add an entry to the TOC section?

Reply:
It turns out that `mapPropsStreamWithConfig` is also missing from the TOC section.

Reply:
Added these functions to the TOC, and also `componentFromStreamWithConfig()`.